spk/imagemagick/Makefile

```diff
@@ -0,0 +1,40 @@
+SPK_NAME = imagemagick
+SPK_VERS = 6.9.0
+SPK_REV = 1
+SPK_ICON = src/imagemagick.png
+DSM_UI_DIR = app
+CHANGELOG =
+
+BETA = 1
+
+DEPENDS = cross/$(SPK_NAME)
+
+MAINTAINER = SynoCommunity
+DESCRIPTION = ImageMagick is a software suite to create, edit, compose, or convert bitmap images.
+RELOAD_UI = yes
+DISPLAY_NAME = ImageMagick
+
+STARTABLE = no
+
+HOMEPAGE = http://www.imagemagick.org/
+LICENSE  = Apache 2.0
+
+INSTALLER_SCRIPT = src/installer.sh
+SSS_SCRIPT       = src/dsm-control.sh
+ADDITIONAL_SCRIPTS =
+
+INSTALL_PREFIX = /usr/local/$(SPK_NAME)
+
+POST_STRIP_TARGET = imagemagick_extra_install
+
+
+include ../../mk/spksrc.spk.mk
+
+.PHONY: imagemagick_extra_install
+ntop_extra_install:
+	install -m 755 -d $(STAGING_DIR)/app
+	install -m 755 -d $(STAGING_DIR)/app/images
+	for size in 16 24 32 48; do \
```

Comment:
Although, now that I'm thinking of it, you don't actually need the DSM_UI_DIR, right? It's a non-startable package, so there's no menu entry needed.

sidenote: we should probably check that sometime...I'm quite sure I add it for packages that don't need it myself...